packages/terra-form/tests/nightwatch/select/PopulatedSelectControlled.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import Select from '../../../lib/Select';
+
+class ControlledSelect extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      animal: 'micros',
+    };
+
+    this.updateAnimal = this.updateAnimal.bind(this);
+  }
+
+  updateAnimal(e) {
+    this.setState({
+      animal: e.target.value,
+    });
+  }
+
+  render() {
+    return (
+      <form>
+        <Select
+          name="foo"
+          options={[{ value: 'puppies', display: 'Puppies' },
+                    { value: 'kittens', display: 'Kittens' },
+                    { value: 'micros', display: 'Microprocessors' },
+                    { value: 'snaps', display: 'Snappers' }]}
+          onChange={this.updateName}
+          value={this.state.animal}
+        />
+        <button type="submit" />
```

Comment:
Give button a label. Also the follow error occurs when the button is clicked:

> warning.js:35 Warning: Failed form propType: You provided a `value` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultValue`. Otherwise, set either `onChange` or `readOnly`. Check the render method of `Select`.